src/gui/LfoControllerDialog.cpp

```diff
@@ -154,7 +131,7 @@ LfoControllerDialog::LfoControllerDialog( Controller * _model, QWidget * _parent
 	moog_saw_wave_btn->setInactiveGraphic(
 		embed::getIconPixmap( "moog_saw_wave_inactive" ) );
 	ToolTip::add( moog_saw_wave_btn,
-			tr( "Click here for a moog saw-wave." ) );
+			tr( "Moog saw wave" ) );
```

Comment:
I think I saw "Moog-like saw wave" somewhere else. We should probably choose one or the other & be consistent throughout.

Reply:
I think "moog-like" is better than just "moog".

Reply:
I noticed that there's a "moog" filter though, so I'll keep it like that.